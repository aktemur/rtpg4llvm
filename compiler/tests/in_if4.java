/*stmt*/
if(true) 
    if(false)
	x = 1;
    else 
	x = 2;
