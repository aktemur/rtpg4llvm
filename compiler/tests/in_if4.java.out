new IfElseStmt(mkConst(true),
      new IfElseStmt(mkConst(false),
      new ExprStmt(mkBinOp(Equal,
         new VarExpr("x"),
         mkConst(1))),
      new ExprStmt(mkBinOp(Equal,
         new VarExpr("x"),
         mkConst(2)))))
